### PR TITLE
Import code initially

### PR DESCRIPTION
**Pull request by nmaludy**

### CODE REVIEW

**Comment by nmaludy** [comment]
@bishopbm1 build is passing now

File: templates/config_ipa_ldap.sh.erb [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (paxri01) [inline templates/config_ipa_ldap.sh.erb:1]: Convert these to epp?

**Review by bishopbm1 — APPROVED** [review]
LGTM

File: manifests/install/client.pp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  if $automount_home_dir != undef {
+    exec { 'client_create_automount_home_dir':
+      command => "mkdir -p ${automount_home_dir}",
+      path    => ['/bin', '/usr/bin'],
+      creates => $automount_home_dir,
+    }
+
+    # Update nsswitch if autofs enabled.
```
Comment (paxri01) [inline manifests/install/client.pp:76]: Need to also check for `sudoers:  files sss`

**Review by paxri01 — APPROVED** [review]
Would like to do some unit tests, but looks good.  Perhaps convert .erb to .epp.  Check nsswitch.conf `sudoers:` line.

**Comment by nmaludy** [comment]
@paxri01 @bishopbm1 FYI i'm removing ipa::client::install::manual as it doesn't function right and isn't used anyways

File: manifests/install/client.pp (hunk 2 above)
Comment (nmaludy) [reply manifests/install/client.pp:111]: what do you want to change here?

Comment (nmaludy) [reply manifests/install/client.pp:109]: done.

**Comment by nmaludy** [comment]
@bishopbm1 @paxri01 this is ready to go

**Review by bishopbm1 — APPROVED** [review]
Looks great thanks

**Pull request merged**